types/errors/link.go

```diff
@@ -0,0 +1,11 @@
+package errors
+
+const (
+	linkCodespace = "link"
```

Comment:
Why is the codespace `link`?

Reply:
Because it was "link" in daphne. For the backward compatibility, we should use the same codespace.